src/Mayaqua/CMakeLists.txt

```diff
@@ -19,11 +19,24 @@ find_package(ZLIB REQUIRED)
 # In some cases libiconv is not included in libc
 find_library(LIB_ICONV iconv)
 
-add_subdirectory(cpu_features)
+if (HAVE_SYS_AUXV)
```

Comment:
Please keep the same style across the `CMakeLists.txt` files (no space before the parenthesis).